render-wasm/src/shapes/strokes.rs

```diff
@@ -46,10 +46,18 @@ pub struct Stroke {
     pub style: StrokeStyle,
     pub cap_end: StrokeCap,
     pub cap_start: StrokeCap,
-    pub kind: StrokeKind,
+    kind: StrokeKind,
 }
 
 impl Stroke {
+    pub fn kind(&self, is_open: bool) -> StrokeKind {
+        if is_open {
+            StrokeKind::CenterStroke
+        } else {
+            self.kind.clone()
+        }
```

Comment:
As commented above, we might not need to check for `is_open` here

Reply:
I think we still need it, Inner and Outer strokes behave as Center ones if they are open, may be it would be more clear calling this method to something like drawing_kind ?

Reply:
I renamed the function to render_kind and added some comments :)